src/actions/util.js

```diff
@@ -0,0 +1,7 @@
+export function keysToVals(obj) {
+  return Object.keys(obj).reduce((acc, key) => (acc[key] = key) && acc, {});
+}
```

Comment:
This allows us to create maps of string-constants conveniently.
Object syntax of...
```js
{
    FOO: null,
    BAR: null,
}
```
...ensures each key is unique, and that IDEs (like IntelliJ) can autocomplete the keys for us in code. Best to extract this to function because there's nothing more embarrassing than running into something like this when you're hard-coding...
```js
{
    FOO: 'FOO',
    BAR: 'FOO', // copypasta accident
}
```